sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md

```diff
@@ -37,6 +37,11 @@ In order to interact with Azure Image Analysis, you'll need to create an instanc
 class. To configure a client for use with Azure Image Analysis, provide a valid endpoint URI to an Azure Computer Vision resource
 along with a corresponding key credential authorized to use the Azure Computer Vision resource.
 
+```C# Snippet:ImageAnalysisUsing
+using Azure.AI.Vision.ImageAnalysis;
+using System;
+using System.IO;
+```
```

Comment:
What about a using statement to support Uri and  AzureKeyCredential?

Reply:
Doesn't ImageAnalysisResult, VisualFeatures.Caption, DetectedTextXxx and ImageAnalysisOptions require using of .Models?

And what about BinaryData?